homeassistant/components/media_player/services.yaml

```diff
@@ -184,3 +184,22 @@ repeat_set:
             - "off"
             - "all"
             - "one"
+
+join:
+  description:
```

Comment:
Please add a `name` item too.

Reply:
OK, I have updated the service description.